Assets/Prefabs/Bushfire.prefab

```diff
@@ -89,7 +89,7 @@ SpriteRenderer:
   m_LightmapParameters: {fileID: 0}
   m_SortingLayerID: 0
   m_SortingLayer: 0
-  m_SortingOrder: 4
+  m_SortingOrder: 3
```

Comment:
Fix fire > dodo

Reply:
Due to collilders the dodo can never be behind the bush anyway so don't need complex logic